Makefile

```diff
@@ -34,6 +34,14 @@ binary: ## build executable for Linux
 cross: ## build executable for macOS and Windows
 	./scripts/build/cross
 
+.PHONY: windows
+windows: ## build executable for Windows
```

Comment:
small nit: how about `binary-windows` and `binary-osx` in both Makefiles?

Reply:
@dnephin changed